addons/report_intrastat/report/report_intrastat_invoice_template.xml

```diff
@@ -38,54 +38,93 @@
             </div>
 
             <!-- Is there a discount on at least one line? -->
-            <t t-set="display_discount" t-value="any([l.discount for l in o.invoice_line_ids])"/>
+            <t t-set="display_discount" t-value="any([line.discount for line in o.invoice_line_ids])"/>
             <!-- show only in sale invoice -->
             <t t-set="type_is_out" t-value="o.type == 'out_invoice'"/>
             <!-- if Rounding Method is computed per line -->
             <t t-set="display_taxamount"
                t-value="o.company_id.tax_calculation_rounding_method == 'round_per_line'"/>
 
+            <!-- TODO ideally: inherit this table from the invoice report -->
```

Comment:
This is a good concern.

But this inherited view is so close from the parent view that it's just more issues to have a duplicate, because it needs to be in sync and we might forget to update it or do it badly. (like you mention just below where the subtotal is not shown...)

Also it serves such a specific legal purpose that it probably won't be heavily customized.

Reply:
However the invoice might be customized and we might not want to customize this report... So I'll remove the TODO but we can keep it in mind.